site/src/pages/components/link-area-delegation-explainer.mdx

```diff
@@ -0,0 +1,191 @@
+---
+menu: Active Proposals
+name: Link Area Delegation (Explainer)
+##path: /components/...
+##pathToResearch: /components/...
+layout: ../../layouts/ComponentLayout.astro
+---
+
+- Authors and contributors: [@noamr](https://github.com/noamr), [@scottaohara](https://github.com/scottaohara), [@nmn](https://github.com/nmn), [@jakearchibald](https://github.com/jakearchibald)
+- Last updated: October 16, 2024
+- Feedback: https://github.com/openui/open-ui/issues/
+
+{/* START doctoc generated TOC please keep comment here to allow auto update */}
+{/* DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE */}
+
+## State of this explainer
+We are currently incubating and exploring a new idea. Chime in!
+
+## Overview
+Historically and for very good reasons, interactive elements on the web cannot be nested within other interactive elements, including hyperlinks.
+Interactive elements not allowing descendent interactive elements is a well established rule in HTML for various interactive elements - where descendent interactive elements or elements with a `tabindex` attribute are specifically called out as invalid (see [the `a` element's content model](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element). Additionally, the [ARIA in HTML guideline]([https://w3c.github.io/html-aria/](https://w3c.github.io/html-aria/#allowed-descendants-of-aria-roles)) specification provides further guidance to discourage the use of interactive elements as descendent of ARIA roles which are themselves considered "widget" (interactive) roles.
+
+Despite this long-standing rule, there are websites which ignore it. In doing so, these sites end up with UI which will be difficult, if not blocking, for people using assistive technology, such as screen readers. On the flip-side, there are also websites such as [Walmart](https://walmart.ca), [Reddit](https://reddit.com), [Rotten Tomatoes](https://rottentomatoes.com), to name a few prominent ones, that use specific markup patterns and CSS or JS to present content that visually appears to have nested targets, as the larger container appears to be clickable - but inspecting the markup patterns would show otherwise.
```

Comment:
```suggestion
Despite this long-standing rule, there are websites which ignore it. In doing so, these sites end up with UI which will be difficult, if not unusable, for people using assistive technology such as screen readers. On the flip-side, there are websites that use CSS and/or JS creatively to create UI that *appears* to have nested interactive elements while avoiding nesting them in the markup itself in order to keep the UI accessible. Examples of such websites include [Walmart](https://walmart.ca), [Reddit](https://reddit.com), and [Rotten Tomatoes](https://rottentomatoes.com) among others.
```

Reply:
Do we need a more detailed write for the "motivation" for this feature?

Reply:
Probably should. But I didn’t have time to get into that. That was another one of my comments that I forgot to repost from my review that didn’t get posted since the explainer got merged while I was reviewing

Reply:
re-reading, i realize maybe you meant that comment just in context to this section.  i'm pulling in some of your ideas / adjusting further.

i'm also changing out the rotten tomatoes example for amazon and best buy, since i could easily find this pattern on both those sites, and still wasn't quite sure where on rotten tomatoes to look.